components/ApplicationTable.tsx

```diff
@@ -104,21 +133,25 @@ const ApplicationTable: FC<ApplicationTableProps> = ({ applications, reviewerIds
   }
 
   return (
-    <>
+    <QueryClientProvider client={queryClient}>
       <Card>
         <Flex gapX="5">
-          <FilterDropdown
-            values={[ALL_DROPDOWN_OPTION, ...Object.keys(NextAction)]}
-            currentValue={nextActionFilterValue}
-            onValueChange={(value) => onFilterDropdownChange(SEARCH_PARAM_NEXT_ACTION, value)}
-            title="Next Action"
-          />
-          <FilterDropdown
-            values={[ALL_DROPDOWN_OPTION, ...reviewerIds]}
-            currentValue={reviewerFilterValue}
-            onValueChange={(value) => onFilterDropdownChange(SEARCH_PARAM_REVIEWER, value)}
-            title="Reviewer"
-          />
+          <Flex gapX="2" align="center">
+            <Text>Next Action: </Text>
+            <Dropdown
+              values={[ALL_DROPDOWN_OPTION, ...Object.keys(NextAction)]}
+              currentValue={nextActionFilterValue}
+              onValueChange={(value) => onFilterDropdownChange(SEARCH_PARAM_NEXT_ACTION, value)}
+            />
```

Comment:
Could we have a reusable `LabelledDropdown` that takes the same props as Dropdown + a label?

Reply:
Is this still necessary after the refactor in [66d5a40](https://github.com/imperial/ug-admissions/pull/6/commits/66d5a4091670cda4acea0daf16b68da64da1399a) to use a `LabelledFormField`?

Reply:
Perhaps not, you are right. We can use a `LabelledFormField` although the name is now out of context, so we need something more generic. Something like `LabelledInput`?

Reply:
We just have a text label component that adds a label to its `children`. See dde94454f4d3791328e1e65607d8f4eaa7cfea5d